streaming/base/util.py

```diff
@@ -18,3 +20,26 @@ def get_list_arg(text: str) -> List[str]:
         List[str]: Splits, if any.
     """
     return text.split(',') if text else []
+
+
+def is_file_exist(filename: str, sleep_time: float, timeout: float, err_msg: str) -> None:
```

Comment:
sleep_time -> poll_interval?

Reply:
Done in https://github.com/mosaicml/streaming/pull/103/commits/bb6649673cbd20eeec770ef30d57668b3ed8afab.